src/main/java/technology/tabula/TextChunk.java

```diff
@@ -62,7 +152,7 @@ public String getText() {
         for (TextElement te: this.textElements) {
             sb.append(te.getText());
         }
-        return sb.toString();
+        return Normalizer.normalize(sb.toString(), Normalizer.Form.NFKD);
```

Comment:
@jazzido So we have to "normalize" the text which, among other things replaces Arabic presentation forms with their standalone forms (like decomposing ligatures). It's gotta go somewhere and it's gotta operate on text, not arrays of TextElements. Do you think this is an appropriate place to put it?

You have a better 30,000-foot view of this library so you may know of a better place to put it.

I should note that it breaks a few tests over two apparently different versions of the letter with an accent. I'll solve that next. What a rabbit hole.

Compare:

```
María 
María 

Cívico
Cívico
```